plugins/modules/cargo.py

```diff
@@ -139,7 +153,8 @@ def get_installed(self):
         cmd = ["install", "--list"]
         data, dummy = self._exec(cmd, True, False, False)
 
-        package_regex = re.compile(r"^([\w\-]+) v(.+):$")
+        package_regex = re.compile(r"^([\w\-]+) v(.+?)( \(registry `.+`\))?:$")
```

Comment:
How about
```suggestion
        package_regex = re.compile(r"^([\w\-]+) v(.+?)(?: \(registry `(.+)`\))?:$")
```
so you can get the registry for the package as `package_info.group(3)` below?